config/envs/default.py

```diff
@@ -118,6 +118,12 @@ class DefaultConfig(object):
     # Post-award submit
     POST_AWARD_SUBMIT_HOST = environ.get("POST_AWARD_SUBMIT_HOST", "")
 
+    # Form Designer
+    FORM_DESIGNER_HOST = environ.get("FORM_DESIGNER_HOST", "")
+
+    # Fund Application Builder
+    FUND_APPLICATION_BUILDER_HOST = environ.get("FUND_APPLICATION_BUILDER_HOST", "")
```

Comment:
Maybe not strictly your fault/problem, but I think that having default values that are invalid isn't super useful 🤔 

I tend to think we should hard error if required configuration isn't available on startup, but maybe we want to tackle that separately...